charts/policy-reporter/charts/policy-reporter-ui/templates/deployment.yaml

```diff
@@ -5,14 +5,27 @@ metadata:
   labels:
     {{- include "ui.labels" . | nindent 4 }}
 spec:
-  replicas: 1
```

Comment:
more than 1 replica make no sense because the functionality is not scalable right now. If you start more then one replicate you have 2 instances watching and report the same informations twice.

Reply:
yep, i understood, but for example some people uses `helm` to disable replicas in `deployment`
because somebody don't use `kubectl scale deployment ....`

Reply:
okay, thats right. It's fine